### PR TITLE
chore: remove unused files

### PR DESCRIPTION
**Pull request by shoom3301**
# Summary

While researching which images should be configurable in the widget mode, I've found that many images are not in use at all

### CODE REVIEW

File: libs/permit-utils/vite.config.ts
Comment (alfetopito) [inline libs/permit-utils/vite.config.ts:1]: This seems unrelated

File: libs/assets/src/images/santa-hat.png
[the remark this answers is not in the document]
Comment (shoom3301) [reply libs/assets/src/images/santa-hat.png:1]: I haven't found it usage in the project, even in the commented code

**Review by alfetopito — APPROVED** [review]

**Review by anxolin — COMMENTED** [review]
Curious. Did you use a tool to find them, or u checked them manually

**Comment by shoom3301** [comment]
> Curious. Did you use a tool to find them, or u checked them manually

Semi-manually, I've used IDEA's tool:

image

**Comment by anxolin** [comment]
Thanks for sharing, that's cool
